### PR TITLE
Provide default for -e for galaxies

### PR DESCRIPTION
**Pull request by philipcmonk**

### CODE REVIEW

File: vere/dawn.c [hunk 1]
```diff
@@ -328,7 +328,7 @@ u3_dawn_vent(u3_noun seed)
   //
   c3_c* url_c = ( 0 != u3_Host.ops_u.eth_c ) ?
     u3_Host.ops_u.eth_c :
-    "https://ropsten.infura.io/v3/196a7f37c7d54211b4a07904ec73ad87";
+    "https://mainnet.infura.io/v3/196a7f37c7d54211b4a07904ec73ad87";
```
[the remark this answers is not in the document]
Comment (philipcmonk) [reply vere/dawn.c]: If we default it in main.c, then it'll flow into %jael state, which we don't want (we want it to ask its parent, and infura doesn't work with %jael anyway).  We only want this default for preboot validation, which is exclusively in dawn.c.

[the remark this answers is not in the document]
Comment (belisarius222) [reply vere/dawn.c]: ok, makes sense

**Comment by vvisigoth** [comment]
screenshot 2019-01-11 17 01 04

**Comment by vvisigoth** [comment]
@philipcmonk

**Comment by philipcmonk** [comment]
Thanks!

**Pull request merged**